spec/unit/provider/package/aix_spec.rb

```diff
@@ -23,15 +23,25 @@
   context "when installing" do
     it "should install a package" do
       expect(@provider).to receive(:installp).with('-acgwXY', '-d', 'mysource', 'mypackage')
+      allow(@provider).to receive(:query).and_return({:name => 'mypackage', :ensure => 'present', :status => :committed})
```

Comment:
From a test flow point of view, mocks and stubs must be before the actual expect, is easier to read and understand the test.